### DISCUSSION TITLE
Adding Restricted Dataset for RF Calculation

### DISCUSSION THREAD

**Discussion by pinarbap** [xumi1993/seispy#73] [General]
Hi everyone,

I'm currently looking into the feasibility of integrating a restricted seismic dataset, with three-component waveforms, into SeisPy alongside the existing FDSN data. I'm seeking input on whether it's viable to introduce a restricted dataset prior to advancing to the RF calculation step.

Thanks

**Comment by xumi1993** [comment]
Do you mean this function? https://seispy.xumijian.me/usage/rf-from-ws.html

Actually, in the ver 1.3.5 (the `dev` branch), you may include following section in the `cfg` file and calculate RFs via `prf` command.

```
[fetch]
# Whether download waveform data from web-service.
use_remote_data = True

# data servers
data_server = IRIS

# username and password on the server, defaults to None
data_server_user =
data_server_password =

# station information
network = CB
station = NJ2
channel = BH?
location = 00
```

But there may still be bugs in this function, because I don't have any account of FDSN service. i will be very grateful that you could help to test this function.

**Reply by pinarbap** [reply]
I have two type of dataset. One is available through FDSN which is fine and easy to calculate RFs. The other dataset which is restricted via FDSN but I have an access and already downloaded the waveforms for each teleseismic event and stored in a local folder. I want to calculate RFs for those waveforms. Yes, I can help if it is possible.
